src/index.php

```diff
@@ -1,3 +1,66 @@
 <?php
-echo "welcome at cronbeat";
+
+// Define application root directory
+define('APP_DIR', dirname(__DIR__));
+
+// Include class files
+require_once APP_DIR . '/src/classes/database.php';
+require_once APP_DIR . '/src/classes/ui.php';
+require_once APP_DIR . '/src/views/base.view.php';
+require_once APP_DIR . '/src/views/setup.view.php';
+require_once APP_DIR . '/src/views/login.view.php';
+require_once APP_DIR . '/src/views/dashboard.view.php';
+require_once APP_DIR . '/src/controllers/BaseController.php';
+require_once APP_DIR . '/src/controllers/SetupController.php';
+require_once APP_DIR . '/src/controllers/LoginController.php';
+require_once APP_DIR . '/src/controllers/DashboardController.php';
```

Comment:
@jetbrains-junie cleanup imports: Just import what's needed here. Move ther imports to the files, where they are then needed.